README.md

```diff
@@ -21,17 +21,59 @@ https://opensource.org/licenses/BSD-3-Clause for full license texts).
 
 Open CAS uses Safe string library (safeclib) that is MIT licensed.
 
+## Installation
+
+To download latest Open CAS Linux release run following commands:
+
+```
+wget https://github.com/Open-CAS/open-cas-linux/releases/download/v19.9/open-cas-linux-v19.9.tar.gz
+tar -xf open-cas-linux-v19.9.tar.gz
+cd open-cas-linux-v19.9
+```
+
+Alternatively, if you want recent development (unstable) version, you can clone GitHub repository:
+
+```
+git clone https://github.com/Open-CAS/open-cas-linux
+cd open-cas-linux
+git submodule update --init
+```
+
+To configure, build and install Open CAS Linux run following commands:
+
+```
+./configure
+make
+make install
+```
+
+The `./configure` performs check for dependencies, so if some of them are missing,
+command will print their names in output. After installing missing dependencies
+you need to run `./configure` once again - this time it should succeed.
+
 ## Getting Started
 
-To quickly install and deploy Open CAS Linux in your system please follow the instructions
-available [here.](https://open-cas.github.io/getting_started_open_cas_linux.html)
+To quickly deploy Open CAS Linux in your system please follow the instructions
+available [here](https://open-cas.github.io/getting_started_open_cas_linux.html).
+
+## Documentation
+
+The complete documentation for Open CAS Linux is available in the
+[Open CAS Linux Administration Guide](https://open-cas.github.io/guide_introduction.html).
+
+## Running Tests
 
-## Guide
+Before running tests make sure you have a platform with at least 2 disks (one for cache and one for core). Be careful as these devices will be most likely overwritten with random data during tests. Tests can be either executed locally or on a remote platform (via ssh) specified in the dut_config.
 
-The complete documentation for Open CAS Linux is available in the Open CAS Linux Administration Guide
-available [here.](https://open-cas.github.io/guide_introduction.html)
+1. Go to test directory `cd test/functional`.
+1. Install dependencies with command `pip3 install -r test-framework/requirements.txt`.
+1. Create DUT config. See example [here](test/functional/config/example_dut_config.yml).  
+    a) Set disks params. You need at least two disks, of which at least one is an SSD drive.  
+    b) For remote execution uncomment and set the `ip`, `user` and `password` fields.  
+    c) For local execution just leave these fields commented.
+1. Run tests using command `pytest-3 --dut-config=<CONFIG>` where `<CONFIG>` is path to your config file, for example `pytest-3 --dut-config="config/dut_config.yml"`.
```

Comment:
Wrong enumeration.

Reply:
This is not a bug, this is a feature. This way you can tell Markdown to use autonumbering, so that if you add new item in the middle (or at the very beginning) of the list you don't need to manually update numbers of all the following items.